cmd/lsjson/lsjson.go

```diff
@@ -75,6 +80,8 @@ If --hash is not specified the the Hashes property won't be emitted.
 
 If --no-modtime is specified then ModTime will be blank.
 
+If --encypted is not specified the Encrypted will be blank.
```

Comment:
--encrypted is spelled wrong here, and if we do the

    `json:",omitempty"`

thing then it should say "won't be emitted." instead of "blank."